tests/regression/fn/fn_cartesian_vertical_advection.cpp

```diff
@@ -27,7 +28,7 @@ namespace {
 
     struct u_forward_scan : fwd {
         static GT_FUNCTION constexpr auto prologue() {
-            return tuple(scan_pass(
+            return nvcc_workarounds::make_1_tuple(scan_pass(
```

Comment:
in tests, our strategy is usually to not #ifdef-enable the workarounds, but just keep it for all compilers.